### ISSUE TITLE
web.xml security constraint for assets folder

### ISSUE THREAD

**Issue by jgambarios** [dotCMS/core#17835]
As part of a series of security improvements we need to add a security constraint for the `assets` folder in the `web.xml` file.

```
<security-constraint>
        <web-resource-collection>
            <web-resource-name>assets</web-resource-name>
            <url-pattern>/assets/*</url-pattern>
        </web-resource-collection>
        <auth-constraint>
        </auth-constraint>
</security-constraint>
```

---
Related to https://github.com/dotCMS/core/issues/17796

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/17836

**Comment by bryanboza** [comment]
Fixed, tested on release-5.2.4 // Postgres // FF
